commands/orb.js

```diff
@@ -28,16 +30,65 @@ Examples:
 `
   }
 
+
   let temp = argv._;
   temp = temp.filter(x => x !== 'orb');
-  const orbName = temp.join(' ');
+  let orbName = temp.join(' ');
+
+  // check orb name against local static array. 
+  let candidates = matchOrbName(orbName);
+
+  // only process further if matchorbName returned an array
+  if (Object.prototype.toString.call(candidates) === '[object Array]') {
+    if (candidates.length == 1) {
+      // unambiguous match, replace input (autocomplete)
+      orbName = candidates[0];
+    }
+    else if (candidates.length > 1) {
+      // multiple matches, suggest some candidates
+      // prioritise orbs which start with the input term
+      candidates = candidates.sort((a, b) => a.toLowerCase().indexOf(orbName.toLowerCase())
+                                           - b.toLowerCase().indexOf(orbName.toLowerCase()))
+
+      // limit number of suggestions
+      const maxCandidates = 5;
+      let suggestions = candidates.slice(0, maxCandidates);
+
+      // append ellipsis, if not all matches are shown as suggestions
+      if (suggestions.length < candidates.length)
+        suggestions.push('...');
+
+      // create suggestions output
+      suggestions = suggestions.join('\n');
+
+      // just show suggestions, no API call
+      return multipleResultsFoundMessage(orbName, suggestions);
+    }
+  }
 
-  let url = 'https://questland-public-api.cfapps.io/orbs/name/'
+  const url = 'https://questland-public-api.cfapps.io/orbs/name/'
     + encodeURIComponent(orbName);
   const response = await fetch(url);
-  return response.ok ? printOrb(await response.json()) : 'Unable to locate orb.';
+  
+  return response.ok ? printOrb(await response.json()) : noResultFoundMessage(orbName, 'Orb');
```

Comment:
The 'no result' currently throws an error for orbs (unexpected end of JSON input), take a look at the public API code to solve